pom.xml

```diff
@@ -491,6 +496,7 @@
         <commons-lang.wso2.version>2.6.0.wso2v1</commons-lang.wso2.version>
         <mavan.findbugsplugin.exclude.file>findbugs-exclude-filter.xml</mavan.findbugsplugin.exclude.file>
         <config.mapper.version>1.0.13</config.mapper.version>
+        <identity.organization.management.version>1.4.45</identity.organization.management.version>
```

Comment:
Bump the version after merging: https://github.com/wso2-extensions/identity-organization-management/pull/392

Reply:
Fixed with https://github.com/wso2/identity-api-user/pull/202/commits/84eb30c0dde1e831268df350241d464ba6b20f7d